bridgeservice/docs/docs.go

```diff
@@ -609,7 +609,7 @@ const docTemplate = `{
                     "example": 42161
                 },
                 "from_address": {
-                    "description": "Address that initiated the bridge transaction",
+                    "description": "Address that initiated the txn on bridge contract. It can be intermediary contract or EOA",
```

Comment:
```suggestion
                    "description": "Address that initiated the transaction on bridge contract. It can be intermediary contract or EOA",
```

I think we should use `transaction` instead of `txn` in docs for the sake of consistency

Reply:
done